module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraServiceAppRoleAssignment.md

```diff
@@ -75,22 +76,22 @@ DeletedDateTime Id                                          AppRoleId
 This example demonstrates how to assign an app role to another service principal in Microsoft Entra ID.
 
 - `ObjectId`:  The ObjectId of the client service principal to which you're assigning the app role.
-- `ResourceId`: The ObjectId of the resource service principal (for example, an API).
-- `Id`: The Id of the app role (defined on the resource service principal) to assign to the client service principal. If no app roles are defined on the resource app, you can use `00000000-0000-0000-0000-000000000000`.
-- `PrincipalId`: The ObjectId of the client service principal to which you're assigning the app role.
+- `ResourceId`: The `Id` of the resource servicePrincipal (the API) that defines the app role (the application permission).
```

Comment:
In this example, the client and resource service principals are the same thing. Is that how we want to bring it out. The ideal scenario would be the client and resource service principals refer to different objects.

Reply:
The test works but is the example to assign app roles from a service principal to itself? Thats my question.